Sources/TABResourceLoader/Services/NetworkDataResourceService.swift

```diff
@@ -38,17 +38,22 @@ open class NetworkDataResourceService<NetworkDataResource: NetworkResourceType &
   }
   
   /**
-   Designated initilizer for NetworkDataResourceService
+   Designated initializer for NetworkDataResourceService
    
    - returns: An new instance
    */
   public required init() {
```

Comment:
`ResourceOperation` requires `.init()` to be defined on the associated type `ResourceService` (conformer to `ResourceServiceType`)

that's why it's there.